include/engine/routing_algorithms/routing_base.hpp

```diff
@@ -636,8 +640,9 @@ template <class DataFacadeT, class Derived> class BasicRoutingInterface
         NodeID middle = SPECIAL_NODEID;
         distance = duration_upper_bound;
 
-        std::vector<std::pair<NodeID, EdgeWeight>> forward_entry_points;
-        std::vector<std::pair<NodeID, EdgeWeight>> reverse_entry_points;
+        using CoreEntryPoint = std::tuple<NodeID, EdgeWeight>;
```

Comment:
any reason to switch to tuple here over pair? Comparison should have been here without the entry_point_comparator already. I feel the access logic for tuple is not that beautiful to read, so I'd prefer to stay at pair.